README.md

```diff
@@ -32,6 +32,9 @@ parameters|sceptre_user_data:
 parameters|sceptre_user_data:
   <name>: !request
     url: <API ENDPOINT>
+    auth: <basic>
```

Comment:
What does this example mean?

Reply:
I think you should include a note that "basic" 
```suggestion
    auth: "basic"
```

Reply:
it means that user can specify multiple auth types (basic, digest, etc..) however only basic is supported at this time.  i will try to clarify

Reply:
done